e2e/benchmarks/browserstack-benchmark/package.json

```diff
@@ -36,7 +36,8 @@
     "build-data": "cd ../../../tfjs-data && yarn && yarn build",
     "build-layers": "cd ../../../tfjs-layers && yarn && yarn build && yarn build-npm",
     "build-vis": "cd ../../../tfjs-vis && yarn && yarn build",
-    "run-cloud-benchmarks": "node app.js --benchmark='./preconfigured_browser.json' --webDeps --cloud --maxBenchmarks=7"
+    "run-cloud-benchmarks": "node app.js --benchmark='./preconfigured_browser.json' --webDeps --cloud --maxBenchmarks=7",
+    "add-firestore": "yarn add firestore"
```

Comment:
You should be able to run `yarn add firebase -D` in your console to add `"firebase": "^8.7.1"` to the dev dependency list. This will automatically add the firestore dependency whenever `yarn` is run, allowing you to remove this line and the accompanying `yarn add-firestore` in line 29.